src/components/AudioTrack/layouts/Full.vue

```diff
@@ -45,9 +45,11 @@ export default {
      * @returns {string} the duration in a human-friendly format
      */
     const timeFmt = (ms) => {
-      const date = new Date(0)
-      date.setSeconds(ms / 1e3)
-      return date.toISOString().substr(11, 8).replace(/^00:/, '')
+      if (ms) {
+        const date = new Date(0)
+        date.setSeconds(ms / 1e3)
+        return date.toISOString().substr(11, 8).replace(/^00:/, '')
+      }
```

Comment:
The return statement from [here](https://github.com/WordPress/openverse-frontend/pull/239/files#diff-7f028a3652b640df123de004df0126bc9acecee3dc7d52a724ceaf52506b6897R79) should be added.

Reply:
Added in [`14e8072`](https://github.com/WordPress/openverse-frontend/pull/239/commits/14e80720abebc9a4479b75affd08b2b9a662a01e).